ReferenceTests/src/tests/examples2d.jl

```diff
@@ -1496,6 +1496,33 @@ end
     fig
 end
 
+@reference_test "Violin" begin
+    fig = Figure()
+    
+    categories = RNG.rand(1:3, 1000)
+    values = RNG.randn(1000)
+    violin(fig[1, 1], categories, values)
```

Comment:
I actually stopped doing reference tests where there's some kind of symmetry in the data. Like here, all groups have the same data basically, so if there was something wrong in the mapping from categories to groups, you wouldn't see it here. Instead I'd add the group values to the data maybe, so you'd see three shifted violins, or something to that effect.